### PR TITLE
Add blog post about API documentation sprint

### PR DESCRIPTION
**Pull request by vpereira**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 9b4c7c53819467d2ad36395b9f391c9e586ea3d7

https://deploy-preview-290--openbuildservice.netlify.app

File: _posts/development/2021-04-12-api-documentation-remake.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Following the last sprint about [API Documentation](/2021/01/26/api-documentation-remake), we focused our last deliverable in documenting some OBS API endpoints with Swagger UI.
+
+In the last days we documented, using OpenAPI, a list of endpoints, which will grow in the next iterations. So here you have:
+
+<figure>
+  <img src="/images/posts/2021-04-12/endpoints.png" alt="OBS API New Documentation Page" />
+  <figcaption>OBS API New Documentation Page</figcaption>
+</figure>
+
+This documentation page is now available [here](https://build.opensuse.org/apidocs-new/).
```
Comment (dmarcoux) [inline _posts/development/2021-04-12-api-documentation-remake.md:16]: Everything after this line is pretty much the same as the previous blog post about the API new documentation. I'm not sure if it's really needed to keep this for every blog post related to API documentation. We could have this in a partial if we decide to keep this. The partials are in https://github.com/openSUSE/obs-landing/tree/master/_includes/partials

Comment (vpereira) [reply _posts/development/2021-04-12-api-documentation-remake.md:16]: i would prefer to drop the two paragraphs:

```
OpenAPI specification is “a specification for machine-readable interface files for describing, producing, consuming, and visualizing RESTful web services” (according to Wikipedia). Once a definition based on the OpenAPI specification is created, a complete ecosystem of tools can be used. One of these tools is Swagger UI, which provides a web interface for the documentation included in the OpenAPI definition.

Swagger UI not only provides an ordered list of the API endpoints, grouped, color-coded by action, examples of parameters and responses, … It also allows you to test your API directly from your browser: asks for authentication when required, asks for the parameters needed, and shows the results of the performed request.
```

File: _posts/development/2021-04-12-api-documentation-remake.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: API Documentation Remake
+category: development
+---
+
+Following the last sprint about [API Documentation](/2021/01/26/api-documentation-remake), we focused our last deliverable in documenting some OBS API endpoints with [Swagger UI](https://swagger.io/tools/swagger-ui/).
```
Comment (dmarcoux) [inline _posts/development/2021-04-12-api-documentation-remake.md:7]:
```suggestion
Following the last deliverable about [API Documentation](/2021/01/26/api-documentation-remake), we focused this time on documenting some OBS API endpoints with [Swagger UI](https://swagger.io/tools/swagger-ui/).
```

File: _posts/development/2021-04-12-api-documentation-remake.md (hunk 1 above)
Comment (dmarcoux) [reply _posts/development/2021-04-12-api-documentation-remake.md:16]: I think it's also better this way.

File: _posts/development/2021-04-12-api-documentation-remake.md (hunk 2 above)
Comment (hennevogel) [inline _posts/development/2021-04-12-api-documentation-remake.md]: Neither sprint nor deliverable makes much sense for our readers. Those are things that only concern us. Just tell them what's new :smile_cat:

File: _posts/development/2021-04-12-api-documentation-remake.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+In the last days we documented, using [OpenAPI specification](https://www.openapis.org/), a list of endpoints, which will grow in the next iterations. So here you have:
```
Comment (dmarcoux) [inline _posts/development/2021-04-12-api-documentation-remake.md:9]: I would drop the link to the OpenAPI specification and just write about the endpoint groups which are now documented.

File: _posts/development/2021-04-12-api-documentation-remake.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: API Documentation Remake
+category: development
+---
+
+Following the last deliverable about [API Documentation](/2021/01/26/api-documentation-remake), we focused this time on documenting some OBS API endpoints with [Swagger UI](https://swagger.io/tools/swagger-ui/).
```
Comment (hennevogel) [inline _posts/development/2021-04-12-api-documentation-remake.md:7]: Neither sprint nor deliverable makes much sense for our readers. Those are things that only concern us. Just tell them what's new :smile_cat:

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**